### PR TITLE
Retry get_queue_url 

### PR DESCRIPTION
**Pull request by garimaarorashuttl**
Issues : 
https://app.clubhouse.io/shuttl/story/3266/stability-error-invalidaccesskeyid
https://app.clubhouse.io/shuttl/story/3257/stability-errors-trackshuttl

### CODE REVIEW

**Comment by garimaarorashuttl** [comment]
Need help in enabling ci

**Comment by rohitpaulk** [comment]
Activated on drone

**Comment by rohitpaulk** [comment]
Closing + opening to see if that triggers CI

File: Changelog.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+## v0.2.19
+
+- Retrying sqs connection if invalid credentials. Background: https://github.com/Shuttl-Tech/monorepo/issues/4084
+
+## v0.2.28
```
Comment (rohitpaulk) [inline Changelog.md]: I think you meant `0.2.18`..

File: simple_worker/queue_providers/sqs_provider.py [hunk 2]
```diff
@@ -39,14 +40,23 @@ def ack(self, queue_name, message_id):
...
         try:
             resp = self.client.get_queue_url(QueueName=full_queue_name)
         except ClientError as e:
-            error_code = "AWS.SimpleQueueService.NonExistentQueue"
-            if e.response["Error"]["Code"] == error_code:
+            if e.response["Error"]["Code"] == "AWS.SimpleQueueService.NonExistentQueue":
                 raise QueueNotFound(full_queue_name)
+            elif e.response["Error"]["Code"] == "InvalidClientTokenId":
+                # retrying coz aws credentials take a while to propagate in all regions:
+                # https://github.com/Shuttl-Tech/monorepo/issues/4084
```
Comment (rohitpaulk) [inline simple_worker/queue_providers/sqs_provider.py]: Link to a publicly available link - `simple_worker` is open source :slightly_smiling_face:

Comment (rohitpaulk) [reply simple_worker/queue_providers/sqs_provider.py]: And the 'retry' comment shouldn't be here, it is should be next to the `@retry` decorator

**Pull request merged**